modApiServer/src/org/aion/api/server/rpc/ApiWeb3Aion.java

```diff
@@ -2872,6 +2838,14 @@ private IRepository getRepoByJsonBlockId(String _bnOrId) {
         return ac.getRepository().getSnapshotTo(b.getStateRoot());
     }
 
+    private AionBlock getBlockByBN(Long bn) {
+        if (bn < 0) {
```

Comment:
Should this not be `bn == -1` instead?

(And if so, I think it'd be good to put -1 in a const like 'BEST_PENDING_BLOCK' or something to that effect)

Reply:
Done